addons/mail/static/src/owl/components/composer_text_input/composer_text_input.js

```diff
@@ -3,9 +3,10 @@ odoo.define('mail.component.ComposerTextInput', function (require) {
 
 const ajax = require('web.ajax');
 const utils = require('web.utils');
+const useStoreCompareKeys = require('mail.hooks.useStoreCompareKeys');
```

Comment:
wrong place

Reply:
Strange to require mail stuff before web stuff but ok to change